src/components/Facility/ConsultationDetails/QuestionnaireResponsesList.tsx

```diff
@@ -237,7 +237,8 @@ function ResponseCard({
   isPrintPreview?: boolean;
 }) {
   const { t } = useTranslation();
-
+  const path = usePath() || "";
+  const isTreatmentSummaryRoute = path?.includes("/treatment_summary");
```

Comment:
This approach of using path to render is not scalable, using a prop would be ideal. In this case we already have prop, isPrintPreview, so we don't have to show a print button when it already a print preview